enroll_batch_students.py

```diff
@@ -0,0 +1,71 @@
+"""
+Sample of file which should be downloaded from Edx course's Instructor Data Download page with students anonymous ids
+
+is located in edx-adapt/data dir.
+"""
+import csv
+from itertools import islice
+import json
+import os
+import sys
+
+import requests
+
+from config import EDXADAPT, DEFAULT_PROBABILITIES, DEFAULT_SKILLS
+
+
+def get_students_for_enrollment(path_to_file):
+    """
+    Parse csv file with student anonymous data and prepare a list of students for enrollment in edx-adapt.
+    """
+    if not path_to_file:
+        print "Usage is: python enroll_batch_student.py /path/to/file/with/student_ids"
+        sys.exit()
+    elif not os.path.exists(path_to_file):
+        print "File does not exist, please try again"
+        sys.exit()
+    students = []
+    with open(path_to_file) as file_with_ids:
+        raw_students_ids = csv.reader(file_with_ids)
+        for line in islice(raw_students_ids, 2, None):
+            students.append(line[1])
```

Comment:
Instead of accumulating students in a `students` list you can `yield` each student id.